src/tools/compiletest/src/main.rs

```diff
@@ -196,6 +197,7 @@ pub fn parse_config(args: Vec<String>) -> Config {
     let has_tidy = Command::new("tidy")
         .arg("--version")
         .stdout(Stdio::null())
+        .stderr(Stdio::null())
```

Comment:
Does tidy write to stderr on your machine?

Reply:
Yes, and in fact it should be on everyone's machine. The first 3 lines of `tidy/src/main.rs`:
```rust
let root_path: PathBuf = env::args_os().nth(1).expect("need path to root of repo").into();
    let cargo: PathBuf = env::args_os().nth(2).expect("need path to cargo").into();
    let output_directory: PathBuf =
        env::args_os().nth(3).expect("need path to output directory").into();
```

Thinking about it, this check is actually broken and always false, because `--version` is entirely ignored and tidy just unconditionally panics. This is another change I can back out here and make into its own PR though, as it is unrelated to the function of the json tests.

Reply:
@CraftSpider there are two different tidy commands. One is `src/tools/tidy`, one is https://www.w3.org/People/Raggett/tidy/. This is checking HTML tidy.

Can you revert this change?

Reply:
Interesting. It was definitely invoking the one I pointed to, I was getting the full panic. I'll revert the change and look into why.